pkg/server/nfs/nfs_server.go

```diff
@@ -29,8 +28,18 @@ NFS_Core_Param
     Protocols = 4;
 }
 
-# uncomment to enable debug logging
-# LOG { COMPONENTS { NFS_V4 = FULL_DEBUG; } }
+LOG {
+	Default_Log_Level = INFO;
+
+# 	uncomment to enable debug logging
+#	COMPONENTS { NFS_V4 = FULL_DEBUG; }
+
+	Facility {
```

Comment:
This is good.

Reply:
Yes, local debugging will be an issue.

We can check the init process (pid=1) and then determine the FILE path.
pid=1 is a longhorn-share-manager process => set to /proc/1/fd/1
otherwise, set to /tm/ganesha.log

Reply:
@weizhe0422 ☝️ Could you do this adaptive change suggested by @derekbit ? WDYT

Reply:
Okay, I will discuss with @derekbit and try to implement it.